src/pages/Result.tsx

```diff
@@ -7,8 +7,12 @@ import useFetchDustInfo, { cityGroup } from '../hooks/useFetchDustInfo';
 import { type SidoDust } from '@/type';
 import { FINE_DUST, ULTRA_FINE_DUST } from '@/utils/constants';
 import { useQuery } from '@tanstack/react-query';
+import { Select } from '@chakra-ui/react';
+import { ChangeEvent, useState } from 'react';
+import { CityGroup } from '@/type';
```

Comment:
```suggestion
import { type SidoDust, type CityGroup } from '@/type';
```
또는 
```suggestion
import type { SidoDust, CityGroup } from '@/type';
```
로 작성해서 한 줄로 합칠 수 있을 것 같아용